releases/v20.2.7.md

```diff
@@ -0,0 +1,93 @@
+
+Get future release notes emailed to you:
+
+{% include marketo.html %}
+
+### Downloads
+
+<div id="os-tabs" class="clearfix">
+    <a href="https://binaries.cockroachdb.com/cockroach-v20.2.7.linux-amd64.tgz"><button id="linux" data-eventcategory="linux-binary-release-notes">Linux</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v20.2.7.darwin-10.9-amd64.tgz"><button id="mac" data-eventcategory="mac-binary-release-notes">Mac</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v20.2.7.windows-6.2-amd64.zip"><button id="windows" data-eventcategory="windows-binary-release-notes">Windows</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v20.2.7.src.tgz"><button id="source" data-eventcategory="source-release-notes">Source</button></a>
+</div>
+
+<section class="filter-content" data-scope="windows">
+{% include windows_warning.md %}
+</section>
+
+### Docker image
+
+{% include copy-clipboard.html %}
+~~~shell
+$ docker pull cockroachdb/cockroach:v20.2.7
+~~~
+
+
+### SQL language changes
+
+- The following [cluster settings](cluster-settings.html) were added: 
+  - `sql.defaults.statement_timeout` controls the default value for the statement_timeout session setting.
+  - `sql.defaults.idle_in_transaction_session_timeout` controls the default value for the `idle_in_transaction_session_timeout` timeout setting.
+  - `sql.defaults.idle_in_session_timeout` already existed, but is now a public cluter setting. [#62185][#62185]
+
+### Bug fixes
+
+- Previously, [changefeeds](change-data-capture.html) were checking the `SELECT` privilege on every descriptor associated with targeted tables, including the database and custom types. They now only check `SELECT` on the targeted tables. [#61661][#61661]
+- The names of custom types are no longer sent to Cockroach Labs in telemetry and crash reports. [#61645][#61645]
+- Fixed a bug where some import failures would cause tables to stay `OFFLINE`, when they should have been brought back to `PUBLIC`. [#61480][#61480]
+- Fixed an npe observed with a `SpanFromContext` call in the stack trace. [#61703][#61703]
+- Exporting to userfile locations now works correctly. [#61789][#61789]
+- The `indexdef` column in the `pg_indexes` table would always report that the index belonged to the public schema. Now it correctly reports user-defined schemas if necessary. [#61753][#61753]
+- Previously, when using `SHOW`, the `idle_in_session_timeout` and `idle_in_transaction_session_timeout` settings would incorrectly display the value of the `statement_timeout` setting. This is now fixed. [#61958][#61958]
+- Fixed a bug where random numbers generated as default expressions during [`IMPORT`](import.html) would collide a few hundred rows apart from each other. [#61630][#61630]
+- Previously, using [`EXPLAIN (OPT, ENV)`](explain.html) on a query that referenced a table in a user-defined schema would fail. This is now fixed. [#61890][#61890]
```

Comment:
```suggestion
- Previously, using [`EXPLAIN (OPT, ENV)`](explain.html) would fail when used on a query that referenced a table in a user-defined schema. This is now fixed. [#61890][#61890]
```